src/Configuration/Levels/LevelRulesResolver.php

```diff
@@ -40,6 +43,8 @@ public static function resolve(int $level, array $availableRules, string $method
             $levelRules[] = $availableRules[$i];
         }
 
+        Assert::allIsAOf($levelRules, RectorInterface::class);
```

Comment:
this is not needed, as already validated before filtering, that just append from existing all RectorInterface instance.

Reply:
:+1: